pymc/tests/test_distributions_moments.py

```diff
@@ -318,11 +326,44 @@ def test_gamma_moment(alpha, beta, size, expected):
             np.arange(1, 6),
             np.arange(2, 7),
             (2, 5),
-            np.full((2, 5), np.arange(2, 7) * special.gamma(1 + 1 / np.arange(1, 6))),
+            np.full(
+                (2, 5),
+                np.arange(2, 7) * special.gamma(1 + 1 / np.arange(1, 6)),
+            ),
         ),
     ],
 )
 def test_weibull_moment(alpha, beta, size, expected):
     with Model() as model:
         Weibull("x", alpha=alpha, beta=beta, size=size)
     assert_moment_is_expected(model, expected)
+
+
+@pytest.mark.parametrize(
+    "n, p, size, expected",
+    [
+        (10, 0.5, None, 5),
+        (10, 0.5, 5, np.full(5, 5)),
+        (10, np.arange(1, 6) / 10, None, np.arange(1, 6)),
+        (10, np.arange(1, 6) / 10, (2, 5), np.full((2, 5), np.arange(1, 6))),
+    ],
+)
+def test_binomial_moment(n, p, size, expected):
```

Comment:
Some of the test combinations should have a `n*p` that is different when rounded and not rounded to make sure it is working as expected